src/benchmarks/micro/coreclr/BenchmarksGame/regex-redux-5.cs

```diff
@@ -26,78 +25,57 @@ namespace BenchmarksGame
     [BenchmarkCategory(Categories.CoreCLR, Categories.BenchmarksGame)]
     public class RegexRedux_5
     {
-        static Regex regex(string re)
-        {
-            // Not compiled on .Net Core, hence poor benchmark results.
-            return new Regex(re, RegexOptions.Compiled);
-        }
-
-        static string regexCount(string s, string r)
-        {
-            int c = 0;
-            var m = regex(r).Match(s);
-            while (m.Success) { c++; m = m.NextMatch(); }
-            return r + " " + c;
-        }
-
         RegexReduxHelpers helpers = new RegexReduxHelpers(bigInput: true);
         
         [Benchmark(Description = nameof(RegexRedux_5))]
-        public int RunBench()
+        [Arguments(RegexOptions.Compiled)]
+        [Arguments(RegexOptions.None)]
+        public int RunBench(RegexOptions options)
         {
             using (var inputStream = new FileStream(helpers.InputFile, FileMode.Open))
             using (var input = new StreamReader(inputStream))
             {
-                return Bench(input, false);
+                return Bench(input, options);
```

Comment:
@adamsitnik, how much of the time in this benchmark is spent reading the input file?  This was all separated out of the benchmark into GlobalSetup in the version you're deleting.

Reply:
I thought that the original benchmark used by The Computer Language Benchmarks Game read it from the file, but it turns out that the input is read from... Console ;)

https://benchmarksgame-team.pages.debian.net/benchmarksgame/program/regexredux-csharpcore-5.html

```cs
var sequences = Console.In.ReadToEnd();
```

I am going to move it to Setup ;)